test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6,13 +6,46 @@
 import unittest
 import sys
 import onnxruntime  # noqa
-
+import torch
+import numpy as np
+import io
 
 class TestONNXRuntime(unittest.TestCase):
 
     def test_onnxruntime_installed(self):
         self.assertTrue('onnxruntime' in sys.modules)
 
+    def test_full(self):
+        class FullModel(torch.nn.Module):
```

Comment:
As I mentioned in another PR, could you refactor the test case bit by creating some helper function.